<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>384</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="scroolLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Select document</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionRoles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomin"/>
    <addaction name="actionZoomout"/>
    <addaction name="actionFit"/>
    <addaction name="actionOrg"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionScan">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Scan directory</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionZoomin">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomout">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="icon">
    <iconset theme="zoom-fit-best">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Fit to screen</string>
   </property>
  </action>
  <action name="actionOrg">
   <property name="icon">
    <iconset theme="zoom-original">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Original size</string>
   </property>
  </action>
  <action name="actionRoles">
   <property name="icon">
    <iconset theme="document-send">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Update roles</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionScan</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>scanClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomFitClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOrg</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOrgClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomInClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRoles</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>rolesClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutClicked()</slot>
  <slot>scanClicked()</slot>
  <slot>zoomInClicked()</slot>
  <slot>zoomOutClicked()</slot>
  <slot>zoomFitClicked()</slot>
  <slot>zoomOrgClicked()</slot>
  <slot>rolesClicked()</slot>
 </slots>
</ui>
